cachi2/core/package_managers/bundler/main.py

```diff
@@ -60,7 +73,10 @@ def _resolve_bundler_package(package_dir: RootedPath, output_dir: RootedPath) ->
     components = [Component(name=name, version=version, purl=main_package_purl.to_string())]
     for dep in dependencies:
         dep.download_to(deps_dir)
-        components.append(Component(name=dep.name, version=dep.version, purl=dep.purl))
+        c = Component(name=dep.name, version=dep.version, purl=dep.purl)
+        if isinstance(dep, GemPlatformSpecificDependency):
+            c.properties = PropertySet(bundler_package_binary=True).to_properties()
```

Comment:
This line actually causes this commit not to pass mypy in a rebase because the attribute is added by the next one.

Reply:
Indeed, thank you!